tasks/configure_nginx.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: copy ssl cert
+  copy:
+    content: "{{ DALITE_SSL_CERT }}"
+    dest: "/etc/ssl/private/ssl-cert.pem"
+    group: www-data
+    mode: "740"
+
+- name: copy ssl key
+  copy:
+    content: "{{ DALITE_SSL_KEY }}"
+    dest: "/etc/ssl/private/ssl-cert.key"
+    group: www-data
+    mode: "740"
```

Comment:
@antoviaque I have double checked, but probably in wrong places (I checked that nginx indeed runs under `www-data`, and is not in any other group). I totally wasn't aware of the whole `ssl-cert` group. 

This is now fixed.